modules/era/sql/era_droplist.sql

```diff
@@ -17134,13 +17109,12 @@ INSERT INTO `mob_droplist` VALUES (2115,0,0,1000,736,@VRARE);   -- Chunk Of Silv
 INSERT INTO `mob_droplist` VALUES (2115,2,0,1000,17296,0);      -- Pebble (Steal)
 
 -- ZoneID: 190 - Rock Eater
-INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,768,@ALWAYS);     -- Flint Stone (Always, 100%)
-INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,640,@VCOMMON);    -- Chunk Of Copper Ore (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,642,@UNCOMMON);   -- Chunk Of Zinc Ore (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,13440,@UNCOMMON); -- Ascetics Ring (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,1027,@RARE);      -- Ranperre Chest Key (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,736,@VRARE);      -- Chunk Of Silver Ore (Very Rare, 1%)
-INSERT INTO `mob_droplist` VALUES (2116,2,0,1000,17296,0);         -- Pebble (Steal)
+INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,768,@ALWAYS);  -- Flint Stone (Always, 100%)
+INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,640,@VCOMMON); -- Chunk Of Copper Ore (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,642,@COMMON);  -- Chunk Of Zinc Ore (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,13440,@RARE);  -- Ascetics Ring (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (2116,0,0,1000,1027,@VRARE);  -- Ranperre Chest Key (Very Rare, 1%)
```

Comment:
http://www.ffxidb.com/zones/190/rock-eater

This seems to be more in line with 5% - 2.4k data at TH0 (3.6%) leads me to believe its higher than 1%. In general it looks like most of the coffer/chest are around 5%

Reply:
Bumped up to 5%